packages/react-avatar/src/components/Avatar/Avatar.types.ts

```diff
@@ -135,6 +135,11 @@ export type AvatarNamedColor =
   | 'platinum'
   | 'anchor';
 
+/**
+ * Sizes that can be used for the Avatar
+ */
+export type AvatarSizes = 20 | 24 | 28 | 32 | 36 | 40 | 48 | 56 | 64 | 72 | 96 | 120 | 128;
```

Comment:
You also probably want to export it from `@fluentui/react-avatar/src/index` and from `@fluentui/react-components`.

Reply:
I will make a follow up PR for the `@fluentui/react-components` part.